tests/std/tests/P0980R1_constexpr_strings/test.cpp

```diff
@@ -613,16 +613,12 @@ constexpr bool test_interface() {
         literal_constructed.shrink_to_fit();
 
         const auto c4 = literal_constructed.capacity();
-        if (is_constant_evaluated()) { // check minimum allocation of _BUF_SIZE when constant evaluated
-            assert(c4 == 16 / sizeof(CharType));
+        if constexpr (is_same_v<CharType, char16_t> || is_same_v<CharType, wchar_t>) {

```

Comment:
Would `sizeof(CharType) == 2` be simpler here?

Reply:
Do you really want to assume the size of character types?

Reply:
This _already_ assumes the sizes of the character types; `7`, `3`, and `15` are not calculated.

Reply:
I wouldn't mind seeing this replaced by `assert(c4 == 16 / sizeof(CharType) - 1);`.

Reply:
👍

Reply:
I'll make this change with the other tiny things I'll push.